client/lib/signup/step-actions/index.js

```diff
@@ -151,6 +151,7 @@ export const getNewSiteParams = ( {
 		public: launchAsComingSoon ? Visibility.PublicNotIndexed : Visibility.PublicIndexed,
 		options: {
 			designType: designType || undefined,
+			launchpad_screen: 'new',
```

Comment:
We'll want to consider using an entirely new option to track whether or not a new Launchpad site is being created

Reply:
What would this solve? It would certainly introduce several other problems.

Reply:
See https://github.com/Automattic/wp-calypso/pull/72619#discussion_r1088227257